.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Hmm, maybe we should add this file to the [`.gitignore`](https://git-scm.com/docs/gitignore)?